MathPix/TeXEditorWindow.xaml.cs

```diff
@@ -127,6 +127,11 @@ private void ResetInput()
       Close();
     }
 
+    private void BtnInsertTags_Click(object sender, RoutedEventArgs e)
+    {
+      TeXInput.SelectedText = "[/$][$]";
```

Comment:
Wouldn't that erase the selection, if it isn't empty ? I would expect something like this instead:

```suggestion
      string.IsNullOrEmpty(TeXInput.SelectedText)
        ? TeXInput.SelectedText = "[/$][$]"
        : TeXInput.SelectedText = $"[/$][$]{TeXInput.SelectedText}[/$][$]";
```